verify.sh

```diff
@@ -94,10 +106,14 @@ function main() {
     cluster)
         verify_cluster
         ;;
+    modprobe)
+        module=$2
```

Comment:
I would suggest to keep the consistence of naming inside and outside of `verify_modprobe` function.
I mean here "module" => "modules"